dns/opnblock/src/opnsense/service/templates/OPNsense/opnblock/opnblock.conf

```diff
@@ -0,0 +1,5 @@
+{% if helpers.exists('OPNsense.opnblock.general') and OPNsense.opnblock.general.Enabled|default("0") == "1" %}
+[general]
+whitelist={{ OPNsense.opnblock.general.whitelist|default("/(api.solvemedia.com)/") }}
+blacklist={{ OPNsense.opnblock.general.blacklist|default("http://pgl.yoyo.org/as/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext") }}
```

Comment:
How do plan the usage with multiple lists? I experienced many problems with public lists, every list does their own stuff which is not supported on every daemon.

Just have a look here:
https://github.com/opnsense/plugins/blob/master/dns/bind/src/opnsense/scripts/OPNsense/Bind/dnsbl.sh

I have no idea how Unbound handles blacklists, but the above was needed for BIND RPZ blacklists

Reply:
The script currently can handle multiple scripts, as long as all the scripts are formatted as "host-lists", eg. `0.0.0.0 1493361689.rsc.cdn77.org`. It runs through the lists, grabs the domain name (eg. $2) and puts it in an entire, eg. `1493361689.rsc.cdn77.org A 0.0.0.0` which is appropriate for Unbound. So yeah, it's limited to the basis that the supplied URL's must contain lists in proper host formatting, like https://github.com/StevenBlack/hosts.

I'd like to rewrite this whole shell file eventually.